### PR TITLE
feat: show order data on thank you page

### PR DESCRIPTION
**Pull request by zoltantakacs-born**
Show the details of the order on the thank you page. It was needed a new composable to gather data about the order and new API functions to send request to BigCommerce.

## Related Issue
VBC-392

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: packages/theme/components/OrderProducts.vue [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+  setup(props) {
+    const orderData = useOrderData();
+
+    const order = orderData.mapOrderSummary(props.order);
+
+    onBeforeUnmount(() => {
+      unMapMobileObserver();
+    });
+
+    console.log(order);
```
Comment (scernisevs) [inline packages/theme/components/OrderProducts.vue:115]: hi Zoltan, could we please remove the console log? :)

Comment (dhargitai) [inline packages/theme/components/OrderProducts.vue:115]: Please delete this.

**Review by scernisevs — APPROVED** [review]
LGTM! :) 1 very minor comment about the console log.
Thank you!

**Review by dhargitai — CHANGES_REQUESTED** [review]
Just some minor things, otherwise looks good.

**Review by dhargitai — APPROVED** [review]

File: packages/api-client/src/api/order/getByCart/index.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import { Endpoints, Order } from '../../../types';
+import { getOrderProducts } from '..';
+import { getOrderShippingAddresses } from '../getShippingAddresses';
+import BigCommerceEndpoints from '../../../helpers/endpointPaths';
+import { stringifyUrl } from 'query-string';
+
+export const getOrderByCart: Endpoints['getOrderByCart'] = async (
+  context,
```
[the remark this answers is not in the document]
Comment (zoltantakacs-born) [reply packages/api-client/src/api/order/getByCart/index.ts:8]: I get the comments from the BigCommerce documentation, they don't give for those 😄

File: packages/api-client/src/types/order/shippingAddress.ts [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  base_handling_cost?: string;
+  /**
+   * The handling charge, excluding tax. (Float, Float-As-String, Integer)
+   */
+  handling_cost_ex_tax?: string;
+  /**
+   * The handling charge, including tax. (Float, Float-As-String, Integer)
+   */
+  handling_cost_inc_tax?: string;
+  handling_cost_tax?: string;
```
[the remark this answers is not in the document]
Comment (zoltantakacs-born) [reply packages/api-client/src/types/order/shippingAddress.ts:50]: I get the comments from the BigCommerce documentation, they don't give for those 😄

File: packages/api-client/src/api/order/getByCart/index.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (zoltantakacs-born) [reply packages/api-client/src/api/order/getByCart/index.ts:8]: The param types are identified from the type of the variable (Endpoints['getOrderByCart'])

**Pull request merged**